ios/sdk/WeexSDK/Sources/Component/WXScrollerComponent.m

```diff
@@ -473,13 +473,14 @@ - (void)scrollViewDidEndDragging:(UIScrollView *)scrollView willDecelerate:(BOOL
 {
     [_loadingComponent.view setHidden:NO];
     [_refreshComponent.view setHidden:NO];
+    
     //refresh
-    if (_refreshComponent && scrollView.contentOffset.y + _refreshComponent.calculatedFrame.size.height < _refreshComponent.calculatedFrame.origin.y) {
+    if (_refreshComponent && scrollView.contentOffset.y < 0 && scrollView.contentOffset.y + _refreshComponent.calculatedFrame.size.height < _refreshComponent.calculatedFrame.origin.y) {
         [_refreshComponent refresh];
     }
     
     //loading
-    if (_loadingComponent &&
+    if (_loadingComponent && scrollView.contentOffset.y > 0 &&
```

Comment:
clarify refreshing and loading  when loading event occurence, the offset.y will be positive, the other will be negtive